docs/tls/CERTIFICATES.md

```diff
@@ -0,0 +1,152 @@
+# Self-signed certificates guide
+
+This guide describes the steps required to generate self-signed certificates for use with NGINX Plus and NLK.
```

Comment:
to do......
This docs needs to stand on its own, since it is a unique file.  So the description must be complete and include the Why, even if a similar Why is repeated across other related docs.

Reply:
Expanded the description to include a little more color